torch/testing/__init__.py

```diff
@@ -1,3 +1,14 @@
+__all__ = [
+    # _C
+    "FileCheck",
+    # _comparisons
+    "assert_allclose",
+    "assert_close",
+    "make_tensor",
+    # submodules
+    "_utils",
```

Comment:
Does it make sense to put underscore prefixed modules in the all list?

Reply:
I put this there due to the explicit `from . import _utils`. This import is otherwise unused.

Reply:
> I put this there due to the explicit `from . import _utils`. This import is otherwise unused.

You can use:

```python
from . import _utils as _utils  # noqa: F401
```